src/commands/images/showDanglingImages.ts

```diff
@@ -0,0 +1,8 @@
+
+import { ext } from "../../extensionVariables";
+import { IActionContext } from 'vscode-azureextensionui';
+
+export async function showDanglingImages(context: IActionContext): Promise<void> {
+    const conf: boolean = await ext.context.globalState.get('vscode-docker.images.showDanglingImages') || false;
```

Comment:
Nit: [`get<T>()`](https://code.visualstudio.com/api/references/vscode-api#Memento) has a form which accepts a default value (rather than testing the result).

Reply:
Good tip. I will fix. Thanks.